packages/axelar-wasm-std/src/response.rs

```diff
@@ -0,0 +1,32 @@
+use cosmwasm_std::{from_json, CosmosMsg, Response, StdError, WasmMsg};
+use serde::de::DeserializeOwned;
+
+#[derive(thiserror::Error, Debug)]
+pub enum Error {
+    #[error(transparent)]
+    Std(#[from] StdError),
+    #[error("no wasm msg found")]
+    NotFound,
+    #[error("multiple msgs found in the response")]
+    MultipleMsgsFound,
+}
+
+/// Get a msg wrapped inside a `WasmMsg::Execute` from a `Response`.
+/// If there are no wasm messages or more than one message in the response, this returns an error.
+pub fn inspect_response_msg<T>(response: Response) -> Result<T, Error>
```

Comment:
This is only used in tests? Then why is it placed here?

Reply:
It's used in tests across multiple crates so needs to be in wasm-std. We should consider adding a `testutils` feature for `axelar-wasm-std` for test only code (such as assert macros). But this PR is already quite large so I didn't want to introduce more refactoring here.